ext/node/ops/zlib/mod.rs

```diff
@@ -12,6 +12,7 @@ use std::future::Future;
 use std::rc::Rc;
 
 mod alloc;
+pub mod brotli;
```

Comment:
Doesn't need to be `pub`?

Reply:
ops need to be public to do `brotli::op_asd::decl()`